pkg/virt-controller/watch/vmi/vmi_test.go

```diff
@@ -818,8 +743,7 @@ var _ = Describe("VirtualMachineInstance watcher", func() {
 		})
 
 		It("should not patch the pod's annotations when network annotations generator returns an empty map", func() {
-			vmi := newPendingVirtualMachine("testvmi")
-			vmi.Status.Phase = virtv1.Running
+			vmi := newPendingVMIWithPhase("testvmi", virtv1.Running)
```

Comment:
On a side note: Pending and Running at the same time?

Reply:
Good catch. It would be good to change the function name to `newTestVMIWithPhase`.